### PR TITLE
Fixes proxy validation error formatting

### PR DESCRIPTION
**Pull request by danehans**
This PR fixes a formatting error in the proxy controller that was added in https://github.com/openshift/cluster-network-operator/pull/245.

/assign @squeed @bparees

### CODE REVIEW

File: pkg/controller/proxyconfig/validation.go [hunk 1]
```diff
@@ -34,8 +34,9 @@ func (r *ReconcileProxyConfig) ValidateProxyConfig(proxyConfig *configv1.ProxySp
 		if err != nil {
 			return fmt.Errorf("invalid httpsProxy URI: %v", err)
 		}
-		if scheme != proxyHTTPScheme && scheme != proxyHTTPSScheme {
-			return fmt.Errorf("httpsProxy requires a %q or %s URI scheme", proxyHTTPScheme, proxyHTTPSScheme)
+		// TODO: Add proxyHTTPSScheme when CA support is added.
+		if scheme != proxyHTTPScheme {
+			return fmt.Errorf("httpsProxy requires a %s URI scheme", proxyHTTPScheme)
```
Comment (bparees) [inline pkg/controller/proxyconfig/validation.go]: i don't see a reason not to allow https today.  it will work for passthrough proxies.

Comment (bparees) [reply pkg/controller/proxyconfig/validation.go]: @danehans bump

Comment (danehans) [reply pkg/controller/proxyconfig/validation.go]: @bparees I addressed your comments in my latest commit.

**Comment by bparees** [comment]
/retest

@squeed i think this is an easy approve, unless you want to pick on the use of single quotes vs %q.

**Pull request merged**